### PR TITLE
More resources and improvements

### PR DESCRIPTION
**Pull request by ekgame**
Changes:
- Added relevant resources from [awesome-osu](https://github.com/cl8n/awesome-osu)
- Added acknowledgments to the readme with some of the sources for this list
- Added a link from the readme to the actual resource list
- Added information for how we maintain the list with a call-to-action for outdated links
- Fixed nested list formatting for GitHub (Kuuube's Tablet Sheets and Guides section)
- Added a "Self-improvement" section and moved Shikke's Anti-Mindblock there

### CODE REVIEW

File: content.md [hunk 1]
```diff
@@ -2,18 +2,24 @@ A collection of tools and resources to enhance your gameplay!
 
 -# Is something missing? Contribute to this list on [GitHub](https://github.com/osucord/resources).
 ---
+### Self-improvement
+- [khz's streaming guide](https://docs.google.com/document/d/1xbNwH_vN4O3azBYwua-Z_3GOAI2d-kV_EeE96aiRLBU/edit) - A guide on how to get better at streaming
+- [Shikke's Anti-Mindblock](https://shikkesora.com/downloads) - A tool to quickly flip your monitor upside to help with mind-block
+- [osu!mania Pattern Repository](https://imgur.com/a/replacement-glossary-osu-mania-pattern-repository-qEwP37W) - A glossary of common osu!mania patterns and terms
+---
```
Comment (Ruben172) [inline content.md:9]: this should not be at the top

File: content.md [hunk 2]
```diff
@@ -67,6 +75,7 @@ A collection of tools and resources to enhance your gameplay!
 - [score inspector](https://github.com/darkchii/score-inspector-extension) - A userscript that adds more stats to user profiles on the official osu! website (Extension compatible with the [osu! scores inspector](https://score.kirino.sh/))
 - [osu! preview](https://github.com/JerryZhu99/osu-preview) - Preview a beatmap without downloading it
 - [osu! subdivide nations](https://github.com/Cavitedev/osu-subdivide-nations) - Display the regional flags and names of players on the official osu! website as well as other unofficial sites (Extension compatible with [osu! World](https://osuworld.octo.moe/))
+- [ezpp!](https://github.com/cl8n/awesome-osu?tab=readme-ov-file#installing) - Calculate how much PP a map is worth for a particular score
```
Comment (Ruben172) [inline content.md:78]: incorrect url -> https://github.com/oamaok/ezpp, mention it's a browser extension

**Review by Ruben172 — CHANGES_REQUESTED** [review]

File: content.md (hunk 2 above)
Comment (ekgame) [reply content.md:78]: The repository weirdly decided to link the chrome/firefox addon marketplaces at the bottom of the readme, so I linked directly to the installing section of the page. It's in the browser extensions category, I don't think that needs another mention.

File: content.md (hunk 1 above)
Comment (ekgame) [reply content.md:9]: I think most players start with the question "how do I get better at the game", then worry about peripherals. That's why I added it to the top. Where would you suggest moving it?

File: content.md (hunk 2 above)
Comment (jamesbt365) [reply content.md:78]: But the link you placed *doesn't go anywhere*.

Best solution is to just link the browser extensions directly.

Comment (Ruben172) [reply content.md:78]: what is bro on about none of that is true

Comment (ekgame) [reply content.md:78]: Oh, right, I don't know how that happened

File: content.md (hunk 1 above)
Comment (Ruben172) [reply content.md:9]: considering these are just random, subjective, guides by random players, I don't think it should be near the top at all.. maybe under training tools? but honestly the whole ordering needs work

File: content.md (hunk 2 above)
Comment (ekgame) [reply content.md]: Should be good now

Comment (jamesbt365) [reply content.md]: Recent commit induced a CI failure.

Comment (ekgame) [reply content.md]: Yeah, I the chrome link was redirecting, I just copied the links from readme without checking

Comment (Ruben172) [reply content.md]: please just only link the github, none of the other ones have the platform specific urls and it shouldn't either as I think that should be left up to the maintainer of the project on github

Comment (ekgame) [reply content.md]: Alright, removed the direct links

File: content.md (hunk 1 above)
Comment (Ruben172) [reply content.md:9]: the mania patterns is also not related to self improvement I think but moreso mapping/skill knowledge? should stay in misc

File: readme.md [hunk 3]
```diff
@@ -54,3 +64,12 @@ Your contributions may be rejected if:
 - The resource is not accessible to everyone (i.e. it requires a subscription).
 - The tool is not maintained or is outdated.
 - The tool is not useful or relevant to the community, or it has very few active users.
+
+## Acknowledgments
+
+This list has been mostly compiled by the moderators of [osu! Game Community Discord server](https://discord.gg/osu) and maintained by the [contributors of this repository](https://github.com/osucord/resources/graphs/contributors).
```
Comment (jamesbt365) [inline readme.md:70]: Its more the other way around is it not? Maintained by the mods and contributed to by the server *for* the server.

File: content.md (hunk 1 above)
Comment (ekgame) [reply content.md]: Yeah, I thought it's a really cool resource, but I couldn't find a better place for it. I also don't like the "Misc" category - it feels kind of dismissive to put something there, as if saying "you're not important enough to be properly categorized".

Comment (Ruben172) [reply content.md]: maybe just put it under mapping resources actually (which fwiw should be renamed to just "Mapping". It's rather redundant with the current title)

Comment (jamesbt365) [reply content.md]: Its not a lack of importance its a lack of quantity, if you have other similar resources feel free to split it off under its own header.

Comment (ekgame) [reply content.md]: It's not really a mapping resource, it's a glossary for how both players and mappers communicate about mania. Maybe we can find more of these glossaries for other modes and combine them into their own category

Comment (Ruben172) [reply content.md]: then put it in misc, it's not personal improvement at the very least

Comment (ekgame) [reply content.md]: Alright, moved

File: readme.md (hunk 3 above)
Comment (ekgame) [reply readme.md:70]: Feel free to suggest better wording

Comment (jamesbt365) [reply readme.md:70]: This works better imo
```suggestion
This list is primarily maintained by the staff of the [osu! Game Community Discord server](https://discord.gg/osu), with contributions from our community. The list of users who have directly contributed to this repository can be found [here](https://github.com/osucord/resources/graphs/contributors).
```
Not sure we really need to link the contributors considering they are on the sidebar on the repository anyway.

Comment (jamesbt365) [reply readme.md:70]: Hows the edit?

Comment (ekgame) [reply readme.md:70]: LGTM

**Review by jamesbt365 — APPROVED** [review]

**Pull request merged**